src/prognostic_equations/vertical_diffusion_boundary_layer.jl

```diff
@@ -17,37 +17,89 @@ import ClimaCore.Operators as Operators
 # 1) turn the liquid_theta into theta version
 # 2) have a total energy version (primary goal)
 
-function eddy_diffusivity_coefficient(C_E::FT, norm_v_a, z_a, p) where {FT}
-    p_pbl = FT(85000)
-    p_strato = FT(10000)
-    K_E = C_E * norm_v_a * z_a
-    return p > p_pbl ? K_E : K_E * exp(-((p_pbl - p) / p_strato)^2)
+function compute_boundary_layer_height(Ri_c::FT) where {FT}
+    # Needs a column search given colidx
+    return nothing
+end
+function compute_bulk_richardson_number(
+    θ_v::FT,
+    θ_a,
+    norm_ua,
+    grav,
+    z,
+) where {FT}
+    norm²_ua = norm_ua^2# SurfaceFluxes uses FT(1) gustiness #TODO Add to ClimaParameters?
+    return (grav * z) * (θ_v - θ_a) / (θ_a * norm²_ua + eps(FT))
+end
+function compute_diffusion_coefficient(
```

Comment:
```suggestion
function compute_drag_coefficient(
```
(I think this is more accurate)

Reply:
Also, I wonder if we can get the exchange coefficient from surface fluxes. That might be more physically consistent. We don't have to do this now though.

Reply:
I agree the `C` should be consistent, but may it be still beneficial to keep this formulation for the initial testing? Also, in the case of wanting to have a continuous diffusivity between the value derived from `SurfaceFluxes.jl` and the interior, I'm getting an extra `\kappa` factor in eqn 19 of Frierson. Maybe it's easier to talk offline about this....

Reply:
Sure, that's ok. Just that the dynamics will still not be the same as Frierson et al. and fms (unless it has been changed), as they use a simplified Monin-Obukhov which is physically consistent with this vertical diffusion. So I think it might be closer to their setup in some sense if we choose C to be the drag coefficient from our surface flux. It probably doesn't matter too much, so either way is fine.